assets/app/views/create/fromimage.html

```diff
@@ -18,7 +18,7 @@
                 <div class="clearfix visible-xs-block"></div>
                 <form class="" ng-show="imageStream" novalidate name="form">
                   <div style="margin-bottom: 15px;">
-                    <div class="form-group form-group-lg">
+                    <div class="form-group">
```

Comment:
It made the input slightly larger, which I personally didn't like since it didn't match the URL input below. It also caused zoom on the iPhone even with the font-size change.